### DISCUSSION TITLE
Is there a way to reduce or change the sensitive when pulling down to dismiss?

### DISCUSSION THREAD

**Discussion by mark8044** [fujidaiti/smooth_sheets#158] [Q&A]
Right now, compared to most other Sheet packages (including Wolt) this packages pull down sensitivity seems too sensitive for my taste.  Meaning its too easy to pull it down to dismiss.  It becomes a problem when using a scrollableSheet.   Is there an easy way to modify this sensitivity?

Thank you much

**Comment by fujidaiti** [comment]
> Is there an easy way to modify this sensitivity?

Not yet, but I will expose some properties to tweak the sensitivity of modal sheets.

**Reply by fujidaiti** [reply]
#162
